future_webpage/dgf/models.py

```diff
@@ -21,8 +21,8 @@ class Division(Model):
     https://www.pdga.com/pdga-documents/tour-documents/divisions-ratings-and-points-factors
     """
 
-    id = models.CharField(max_length=10, null=False, blank=False, primary_key=True)
-    text = models.CharField(max_length=100, null=False, blank=False)
```

Comment:
to esto es default

Reply:
<chiste> nononononono, si no lo pones extra manolo se ralla y te dice que la base de datos está mal </chiste> :P hahahahahhaha